### PR TITLE
chore: update GitHub bug report template

### PR DESCRIPTION
**Pull request by Jolg42**
Add databases and code blocks

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -36,15 +36,22 @@ body:
       description: Your Prisma schema, Prisma Client queries, ...
       value: |
         <!-- Do not include your database credentials when sharing your Prisma schema! -->
+        ```prisma
+        // Add your schema.prisma
+        ```
+
+        ```ts
+        // Add your TypeScript code
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/bug_report.yml:44]:
```suggestion
        // Add your code using Prisma Client
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -36,15 +36,22 @@ body:
...
   - type: textarea
     attributes:
       label: Environment & setup
-      description: In which environment does the problem occur
+      description: In which environment does the problem occur?
       value: |
-        - OS: <!--[e.g. Mac OS, Windows, Debian, CentOS, ...]-->
-        - Database: <!--[PostgreSQL, MySQL, MariaDB or SQLite]-->
+        - OS: <!--[e.g. macOS, Windows, Debian, CentOS, ...]-->
+        - Database: <!--[PostgreSQL, MySQL (PlanetScale?), MariaDB, SQL Server, SQLite, MongoDB or CockroachDB]-->
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]:
```suggestion
        - Database: <!--[PostgreSQL, MySQL, PlanetScale, MariaDB, SQL Server, SQLite, MongoDB or CockroachDB]-->
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -1,19 +1,22 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
 name: Bug report
 description: Create a bug report to help us improve Prisma
-labels: 'kind/bug'
+labels: ['kind/bug', 'bug/1-unconfirmed']
```
Comment (Jolg42) [inline .github/ISSUE_TEMPLATE/bug_report.yml:4]: @janpio What do you think about this?

Comment (janpio) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]: See Slack conversation.

Comment (Jolg42) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]:
```suggestion
labels: ['kind/bug']
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -27,28 +30,39 @@ body:
...
       description: Your Prisma schema, Prisma Client queries, ...
       value: |
         <!-- Do not include your database credentials when sharing your Prisma schema! -->
+        ```prisma
+        // Add your schema.prisma
+        ```
+
+        ```ts
+        // Add your code using Prisma Client
+        ```
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/bug_report.yml:50]: Just realizing that this could backfire with a lot of empty code blocks. Let's see how this goes.

**Pull request merged**